test/AccountManager.test.ts

```diff
@@ -98,6 +98,42 @@ describe('AccountManager', function () {
       });
     });
 
+    describe('removeAccount()', function () {
+      let fakeAccounts: Array<Wallet>;
+
+      beforeEach(function () {
+        fakeAccounts = new Array<Wallet>(3).fill(Wallet.createRandom());
```

Comment:
NIT: I'd store 3 in a constant `initialLength` to be used elesewhere also.

Reply:
Done.